app/src/main/java/org/wikipedia/donate/DonorStatus.kt

```diff
@@ -7,6 +7,9 @@ enum class DonorStatus {
 
     companion object {
         fun donorStatus(): DonorStatus {
+            // when donor status is set from developer settings
+            Prefs.donorStatus?.let { return it }
```

Comment:
Not sure if we really need this preference for only testing purpose. Would it be possible to remove this, and let the designer know that you can fill up a `donation result` by typing the date format?

So that we don't have to add a developer preference for testing only.

Reply:
done/